cli.js

```diff
@@ -1,32 +1,64 @@
 #!/usr/bin/env node
 'use strict';
 
+/* eslint no-console: off */
+
 const debug = require('debug')('hmpo:journey-runner:cli');
+
+const _ = require('lodash');
+const fs = require('fs');
+const path = require('path');
+const promisify = require('util').promisify;
+const readFile = promisify(fs.readFile);
+
 const runner = require('./runner');
 
-let argv = require('yargs')
-    .command('--headless', 'Run in headless mode')
-    .command('--journey <config>', 'specify the config', (yargs) => {
-        yargs.positional('config', {
-            describe: 'config file'
-        });
-    })
-    .command('--host <host>', 'specify a host to run again', (yargs) => {
-        yargs.positional('host', {
-            describe: 'host',
-            default: 'http://localhost:9001'
-        });
-    })
-    .argv;
-
-(async function run() {
+async function loadConfig(filename) {
+    filename = path.resolve(filename);
+    let basePath = path.posix.dirname(filename);
+    let fileData = await readFile(filename);
+    let config = JSON.parse(fileData, (key, value) =>
+        typeof value === 'string' && value.startsWith('file://')
+            ? path.join(basePath, value.slice(7))
+            : value
+    );
+    return config;
+}
```

Comment:
load config file as part of the cli interface. If this module is used directly then an object can be supplied instead